lms/djangoapps/course_blocks/transformers/tests/helpers.py

```diff
@@ -115,7 +116,10 @@ def add_parents(self, block_hierarchy, block_map):
             # It would be re-added to the course if the course was
             # explicitly listed in parents.
             course = modulestore().get_item(block_map['course'].location)
-            course.children.remove(block_key)
+            try:
+                course.children.remove(block_key)
+            except ValueError:
+                pass  # block_key was not a direct child of course, carry on
```

Comment:
We should make sure this isn't masking an issue in the test code.  As the original code didn't expect this condition.